design.md

```diff
@@ -0,0 +1,94 @@
+# y2c
+
+## Description
+
+Automatically import events from Github repos into Google Calendar.
+
+## Overview
+
+Github Action that reads event data from specific YAML files (`y2c.yml`) in a GitHub repo and calls the Google Calendar API to create/update events. The action is only triggered when master is pushed and the files containing event data change. Event attendees can either be individual's email addresses or a Google Group email address.
```

Comment:
or a groups.io email list. In groups.io case, the email invite will be sent to the mailing list.

Reply:
@dankohn in that case, adding a new member to group.io won't send an invitation to the new member, right?

Reply:
Yes, exactly. Please make a note of that in the Readme.